spec/services/get_application_forms_waiting_for_provider_decision_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe GetApplicationFormsWaitingForProviderDecision do
+  let(:current_time) { Time.zone.local(2019, 6, 1, 12, 0, 0) }
+
+  around do |example|
+    Timecop.freeze(current_time) do
+      example.run
+    end
+  end
+
+  def create_application(status:, reject_by_default_at:)
+    application_form = create :application_form
+    create(
+      :application_choice,
+      application_form: application_form,
+      status: status,
+      reject_by_default_at: reject_by_default_at,
+    )
+    application_form.reload
```

Comment:
You could return the `application_choice` so we don't have to do `application_form.application_choices.first` elsewhere.